src/Common/tests/System/PlatformDetection.cs

```diff
@@ -207,49 +207,72 @@ private static bool IsDistroAndVersion(string distroId, string versionId = null)
             return false;
         }
 
-        private static IdVersionPair ParseOsReleaseFile()
+        public static string GetDistroVersionString()
+        {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+            {
+                return "";
+            }
+
+            DistroInfo v = ParseOsReleaseFile();
+
+            return "Distro=" + v.Id + " VersionId=" + v.VersionId + " Pretty=" + v.PrettyName + " Version=" + v.Version;
+        }
+
+        private static DistroInfo ParseOsReleaseFile()
```

Comment:
If they use all of the Unix related api then yes you are right. This shouldn't block you from merging, it's just something I noticed. Perhaps the right thing then is to rename the PlatformDetection.Unix.cs file to something like: PlatformDetection.Unix.Interop or something like that.